frontend/src/js/store/organizationSlice/constants.ts

```diff
@@ -60,6 +60,7 @@ export const AUDIT_LOGS_TYPES = [
   { title: 'Device', queryParameter: 'object_id', value: 'device' },
   { title: 'User', queryParameter: 'object_id', value: 'user' }
 ];
+export const SP_AUDIT_LOGS_TYPES = [...AUDIT_LOGS_TYPES, { title: 'Tenant', queryParameter: 'object_id', value: 'tenant' }];
```

Comment:
https://www.figma.com/design/xwdyJV8aKd5J3Rk0lBqzIS/Multi-tenant-access-UX?node-id=128-1501&node-type=instance&t=bx3dX27E0bjE9Y5l-0 shows only the `User` and `Tenant` in the list, this would also allow filtering for entries not relevant to the SP